plugins/idea/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
+    version = '2021.3.3'
```

Comment:
This change to the build was necessary as the 1.0 of the IntelliJ gradle plugin seems to have an issue when working with the latest IntelliJ EAP (which the build downloads so that it can build against it). This problem is documented here:

https://youtrack.jetbrains.com/issue/IDEA-291192

The solution would be to update to plugin version 1.5.2 - but that plugin version still requires an intellij version to be specified manually. I found that by specify the current IntelliJ version as the IntelliJ API we wanna build against, things work just fine, even with the 1.0 plugin.